frontend/src/contexts/HostContext/HostContext.js

```diff
@@ -35,6 +35,17 @@ const initialState = {
 
 export const HostContext = createContext();
 
+function allNamesUppercase(currentState) {
```

Comment:
This sounds like a variable name, not a function name. Functions *do* something.

maybe: "uppercaseAllNamesInState"

Reply:
We could probably use a `some` operation instead to make what this is doing more clear.

Reply:
(Effectively) resolved in 541bd19